system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/Participant.java

```diff
@@ -43,8 +43,13 @@
 import static org.eclipse.edc.junit.testfixtures.TestUtils.getFreePort;
 
 public class Participant {
-
+    public static final String E2E_TEST_NAME;
```

Comment:
I'm not a big fan of these shared "static" variables initialized in a static block.
I think this "test name" variable should belong to the test class, and then it could be passed to the `controlPlaneCosmosDbConfiguration` to set up the configuration and used in the beforeAll to create the database